### PR TITLE
Feat/loading UI

### PR DESCRIPTION
**Pull request by westalert**
add animation for loading UI.

### CODE REVIEW

File: src/legacy/Routes.tsx [hunk 1]
```diff
@@ -336,9 +346,10 @@ class RouteApp extends React.Component<any, any> {
       info: newInfo,
     })
   }
-  setInfo = (newInfo: any) => {
+  setInfo = (newInfo: TWalletInfo) => {
     // If the price is not set in this object, copy it over from the current object
     const { info } = this.state
+    console.log('newInfo', newInfo)
```
Comment (adrianobetkal) [inline src/legacy/Routes.tsx:352]: Redundant console.log

File: src/legacy/Routes.tsx [hunk 2]
```diff
@@ -610,8 +621,9 @@ class RouteApp extends React.Component<any, any> {
 
               <Route
                 path={routes.LOADING}
-                render={() => (
+                render={p => (
```
Comment (adrianobetkal) [inline src/legacy/Routes.tsx:624]: I don't know what `p` is, use more descriptive name

Comment (westalert) [reply src/legacy/Routes.tsx:624]: p: prop of route when rendering @adrianobetkal 
It's so easy for understand for developers.

File: src/legacy/Routes.tsx (hunk 1 above)
Comment (westalert) [reply src/legacy/Routes.tsx:352]: okay, I will remove

File: src/legacy/Routes.tsx (hunk 2 above)
Comment (adrianobetkal) [reply src/legacy/Routes.tsx:623]: The good practice is to never user letters or shorts. Any argument, const, let, etc names should be descriptive even if you think `It's so easy for understand for developers.`

**Review by adrianobetkal — APPROVED** [review]

**Pull request merged**